Templates/navbar.pug

```diff
@@ -7,10 +7,10 @@
             ul.left.hide-on-med-and-down
                 li#nav-projects
                     a.navbarLink(href='#!') #{__('pageNames.projects')}
-                if canAccessUsers
+                if canAccessUsersList
```

Comment:
I dont think this should be changed to `list`, it doesnt reflect the correct behaviour. also done you need `user.canAccess.....`

Reply:
same with setings